prediction.go

```diff
@@ -38,7 +38,7 @@ type PredictionInput map[string]interface{}
 type PredictionOutput interface{}
 
 // CreatePrediction sends a request to the Replicate API to create a prediction.
-func (r *Client) CreatePrediction(ctx context.Context, version string, input PredictionInput, webhook *Webhook) (*Prediction, error) {
+func (r *Client) CreatePrediction(ctx context.Context, version string, input PredictionInput, webhook *Webhook, stream bool) (*Prediction, error) {
```

Comment:
This method is a good candidate for refactoring to use functional option arguments, like #6